lib/concurrent/ivar.rb

```diff
@@ -57,15 +57,13 @@ class IVar
     # @option opts [String] :copy_on_deref (nil) call the given `Proc` passing
     #   the internal value and returning the value returned from the proc
     def initialize(value = NO_VALUE, opts = {})
-      init_obligation
+      super(&nil)
```

Comment:
no need for `&nil` here, it is required only when initialize is called with block which should not be passed to super